
/*public class tryaa_catch {
    public static void main (String args[]){
        int a=5000;
        int b=0;
        try{
            int c=a/b;
        System.out.println("the result is"+c);
        }
        catch(Exception e){
            System.out.println("sorry");
            System.out.println(e);

        }
        
    }
    
}
*/
